doc/README_Source.md

```diff
@@ -2,144 +2,168 @@
 
-### Install Connector Manually
-Download and extract the ZIP file for your connector and follow the manual connector installation [instructions](https://docs.confluent.io/current/connect/managing/install.html#install-connector-manually)
+- [Quickstart](#quickstart)
+- [Source configuration properties](#sink-configuration-properties)
 
-## Configuration
+## Quickstart
 
-At the moment the following settings can be configured by means of the *connector.properties* file. For a config file containing default settings see [this example](../src/integration-test/resources/source.config.json).
+### Prerequisites
+
+- Confluent Platform (recommended to use this [setup](./Confluent_Platform_Setup.md))
+  - If you plan on using a separate Confluent Platform instance, you will need to install the Cosmos DB connectors manually.
+- Cosmos DB Instance ([setup guide](./CosmosDB_Setup.md))
+- Bash shell (tested on Github Codespaces, Mac, Ubuntu, Windows with WSL2)
+  - Will not work in Cloud Shell or WSL1
+- Java 11+ ([download](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html))
+- Maven ([download](https://maven.apache.org/download.cgi))
 
-| Name                                           | Description                                                                                          | Type    | Default                                                                       | Valid Values                                                                                                     | Importance |
-|------------------------------------------------|------------------------------------------------------------------------------------------------------|---------|-------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------|------------|
-| connect.cosmosdb.databasename                             | name of the database to write to                                                              | string  |
-| connect.cosmosdb.master.key | the configured master key for Cosmos DB | string |
-| connect.cosmosdb.connection.endpoint | the endpoint for the Cosmos DB Account | uri |
-| connect.cosmosdb.containers.topicmap | comma separeted topic to collection mapping, eg. topic1#coll1,topic2#coll2 | string
-| connect.cosmosdb.containers | list of collections to monitor | string
-| connect.cosmosdb.changefeed.startFromBeginning | set if the change feed should start from beginning | boolean | true 
-| connect.cosmosdb.task.poll.interval | interval to poll the changefeedcontainer for changes  | int
+### Install source connector
 
-### Kafka Connect Converter Configuration
+If you are using the Confluent Platform setup from this repo, the Cosmos DB Source Connector is included in the installation and you can skip this step. Otherwise, you will need to package this repo and include the JAR file in your installation.
 
```

Comment:
they can pick-up a jar file from the github releases page. easier than rebuilding the jar themselves. 
please include a link to github releases. 
once we get certified and available on Confluent Hub then we need to update this document again to point to the official location for the connectors.

Reply:
Linked the Release page to grab latest JAR and put these instructions in block to refer